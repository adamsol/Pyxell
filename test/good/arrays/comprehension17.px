
print [for i in 1..3 yield i...5]
print [for c in "zux" yield c..'z' by 2].join()
