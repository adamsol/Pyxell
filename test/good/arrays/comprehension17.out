[1, 2, 3, 4, 2, 3, 4, 3, 4]
zuwyxz
